README.md

```diff
@@ -73,147 +73,97 @@ Once you are in the folder that contains the `environemnt.yml` file, execute the
 ```sh
 conda env create -n <env_name> environment.yml
 conda activate <env_name>
-python -m pip install -ve .
+python -m pip install --no-deps -e .
 ```
-Execute the following command if you plan to contribute code:
+Note the use of `--no-deps`. Indeed the `environment.yml` already instructs conda to install all the necessary package dependencies. We therefore prevent `setup.py` from overwriting them with the pip-equivalent packages, which might not be compatible with conda. 
+
+If you plan to contribute code, you need extra packages, which you can install by running the following command instead of (or after) the previous one:
 ```
-python -m pip install -r requirements-dev.txt -ve .
+python -m pip install --no-deps -r requirements-dev.txt -e .
```

Comment:
oh, that is right, I did not think about it.
I thought I created that file, but I clearly didn't, I must be remembering something form the past.
so that, or we can alternatively just have a pip install -r requirements-dev.txt run after the qiskit metal install (one after the other). I can test if that works. But your solution is solid (the one missing might be ok since they do not build docs any longer - can test that via github actions)

Reply:
I opted for removing the `--no-deps` and the `-e .` and moved the code at the bottom of README.md